src/Microsoft.Extensions.Logging.Log4Net.AspNetCore/Log4NetProviderOptions.cs

```diff
@@ -80,5 +80,10 @@ public Log4NetProviderOptions(string log4NetConfigFileName, bool watch)
         /// Gets or sets the scope factory.
         /// </summary>
         public Log4NetScopeFactory ScopeFactory { get; set; }
-    }
+
+        /// <summary>
+        /// Let user setup log4net externally
+        /// </summary>
+        public bool ExternalConfigurationSetup { get; set; } = false;
```

Comment:
The assignment is not required, I think. The default value for a boolean property is already false. Can you remove it?

Reply:
You're correct, the assignment is not required. I'll remove it